zone/spell_effects.cpp

```diff
@@ -8189,6 +8193,1098 @@ bool Mob::PassCastRestriction(int value)
 	return false;
 }
 
+void Mob::SendCastRestrictionMessage(int requirement_id, bool target_requirement, bool is_discipline) {
+
+	if (!IsClient()) {
+		return;
+	}
+	/*
+		Most of these are the live messages that modern clients give. Current live dbstr_us type 39
+		Having these messages display greatly enhances the useabllity of these fields. (CastRestriction=target, caster_requirement_id=caster)
+		If target_requirement is false then use the caster requirement message.
+		Added support for different messages for certain high yield restrictions based on if
+		target or caster requirements.
+
+	*/
+
+	std::string msg = "";
+
+	if (target_requirement) {
+		msg = "Your target does not meet the spell requirements. ";
```

Comment:
Extra space here? The messages below add a space after these.

Reply:
Intentional, because that is combined with the specific message string. That is how live, displays it.

Reply:
Ah okay, so the space here and the additional space added in the Message is intended, got it.